docs/it/refactoring.md

```diff
@@ -0,0 +1,136 @@
+# Refactor
+
+## Che cos'è il refactoring
+
+Alcuni direbbero che il refactoring è l'arte di crearsi problemi.
+Il refactoring è un insieme di attività dedite alla revisione di una o più parti del codice di cui è composta la nostra applicazione.
+Può interessare un singolo metodo, una singola classe, un intero modulo.
+L'obiettivo di queste attività è rivedere la logica oppure la scrittura del codice che la compone, in modo da ottenere migliori performance (o un maggior disaccoppiamento, una maggiore testabilità) senza però, in alcun modo, alterarne il comportamento e, soprattutto, aggiungere nuove funzionalità.
+Se l'attività è stata fatta bene, al termine avremo un codice più pulito, comprensibile, di cui non è stato alterato il comportamento e che passa perfettamente tutti i test già presenti.
+Ma andiamo per gradi.
+
+## Benefici
+
+Uno dei principali benefici che otterremo con il refactoring è la rimozione di porzioni di codice non performanti, sostituiti da codice più trasparente e chiaro. E con questo, un'evidente riduzione del debito tecnico accumulato.
```

Comment:
```suggestion
Uno dei principali benefici che otterremo con il refactoring è la rimozione di porzioni di codice non performanti, sostituiti da codice più trasparente e chiaro. E con questo, un'evidente riduzione del debito tecnico accumulato.
```

Forse la seconda frase ha un typo o comunque qualcosa da correggere, mi pare che una frase non possa cominciare con la E, a parte questa piccola correzione concettualmente ci siamo!

Reply:
Forse manca il plurale (_questo_ si riferisce effettivamente a _benefici_).
Riguardo l'iniziare una frase con una congiunzione, è perfettamente [corretto](https://accademiadellacrusca.it/it/consulenza/sulluso-delle-congiunzioni-allinizio-di-frase/25).